content/en/synthetics/guide/synthetic-test-monitors.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Using Synthetic Test Monitors
+kind: guide
+description: Learn about Synthetic monitors created with your Synthetic tests. 
+further_reading:
+- link: "/monitors/manage/"
+  tag: "Documentation"
+  text: "Manage Monitors"
+- link: "/synthetics/metrics/"
+  tag: "Documentation"
+  text: "Synthetic Monitoring Metrics"
+---
+
+## Overview
+
+When you create a Synthetic test, Datadog automatically creates an associated monitor. You can setup notifications when the Synthetic test monitor alerts.
+
+{{< img src="synthetics/guide/synthetics_test_monitors/synthetic_test_monitor.png" alt="Synthetic test monitor" style="width:100%;">}}
+
+## Create a Synthetic test monitor
+
+<div class="alert alert-info">You cannot create or import a Synthetic test monitor in <a href="/monitors/create/">Monitors</a>.</div>
+
+Create a monitor in the **Configure the monitor for this test** section to send notifications when a Synthetic test is failing. Monitors are associated with the Synthetic test you create and link to the alerting conditions set in your Synthetic test configuration. To use monitor attribute and tag variables, create a [metric monitor][1].
+
+{{< img src="synthetics/guide/synthetics_test_monitors/configure_the_monitor_for_this_test.png" alt="Creating a monitor in your Synthetic test" style="width:90%;">}}
+
+Customize the monitor name to search for it on the [**Manage Monitors**][2] page. To find a Synthetic test monitor, filter on `type:synthetics` in the search bar. You can use monitor [conditional variables][3] to characterize the notification message based on test state. 
+
+The Synthetic test monitor integrates with notification channels such as email, Slack, Pagerduty, and Microsoft Teams. For more information, see [Notifications][4].
+
+If you have multiple layers of notifications (for example, notifying more teams the longer a Synthetic test is alerting), Datadog recommends enabling [renotification][5] on your Synthetic monitors.
+
+## Tailor monitor notifications
+
+Depending on your incident management strategy, you may want to involve multiple teams when a Synthetic test alerts. Add `{{#is_renotify}}` and `{{/is_renotify}` in the escalation message to notify Team A first, and Team B if the issue still occurs. Use [conditional variables][3] to characterize the notification message based on team. 
```

Comment:
```suggestion
Depending on your incident management strategy, you may want to involve multiple teams when a Synthetic test alerts. To notify Team B only on subsequent alerts after the first alert, surround the notification to Team B with `{{#is_renotify}}` and `{{/is_renotify}`. Use [conditional variables][3] to further characterize the notification message based on monitor attributes. 
```
This line is fine as is, but could be reworded to be clearer about how exactly you write the message to notify a different team upon escalation.